src/json/from_json.cr

```diff
@@ -68,7 +68,7 @@ end
 
 {% for type in %w(Int8 Int16 Int32 Int64 UInt8 UInt16 UInt32 UInt64) %}
   def {{type.id}}.new(pull : JSON::PullParser)
-    {{type.id}}.new(pull.read_int)
+    {{type.id}}.new!(pull.read_int)
```

Comment:
Shouldn't this raise on overflow?

Reply:
Not quite. The json lexer tocken has a `int_value : Int64` and the support for big UInt64 values comes from an intermediate negative number stored in `int_value`. So... unless there is a broader refactor we need the `new!`.

Reply:
What about the other types? (`Int32, `Int8`, etc.).

But yes, it worked like that before (no raise on overflow) so once we have overflow working we can probably tackle it separately, so don't worry.

Reply:
This must be commented in the code, or compiled into an issue, etc.

This is rushing.